modules/currency_denomination/includes/commerce_pos_currency_denomination.admin.inc

```diff
@@ -0,0 +1,361 @@
+
+  drupal_set_message(t('The denomination type has been successfully added.'));
+  drupal_goto('admin/commerce/config/pos/denomination');
+}
+
+/**
+ * Delete a denomination type.
+ */
+function commerce_pos_currency_delete_denomination_type_form($form, &$form_state, $currency_id) {
+  // Get the denomination type info.
+  $form_state['denomination_type_info'] = commerce_pos_denomination_get_currency_types_from_id(array($currency_id));
+
+  if (!empty($form_state['denomination_type_info'])) {
+    $form_state['denomination_type_info'] = $form_state['denomination_type_info'][$currency_id];
+
+    return confirm_form(
+      $form,
+      t('Are you sure you want to delete currency denomination type @type?', array(
+        '@type' => $form_state['denomination_type_info']->currency_code
+      )),
+      'admin/commerce/config/pos/denomination',
+      t('All denominations under this type will be deleted. This action cannot be undone.'),
+      t('Delete'),
+      t('Cancel')
+    );
+  }
+  else {
+    form_set_error('', t('Invalid currency ID.'));
+  }
+
+  return $form;
+}
+
+/**
+ * Submit handler for deleting a denomination type.
+ */
+function commerce_pos_currency_delete_denomination_type_form_submit($form, &$form_state) {
+  if (isset($form_state['denomination_type_info'])) {
+    // First delete the denominations for that type.
+    db_delete('commerce_pos_currency_denomination')
+      ->condition('currency_id', $form_state['denomination_type_info']->currency_id)
+      ->execute();
+
+    // Now delete the denomination type.
+    db_delete('commerce_pos_currency_denomination_type')
+      ->condition('currency_id', $form_state['denomination_type_info']->currency_id)
+      ->execute();
+  }
+
+  drupal_set_message(t('The denomination type @type has been successfully deleted.', array(
+    '@type' => $form_state['denomination_type_info']->currency_code
+  )));
+  drupal_goto('admin/commerce/config/pos/denomination');
+}
+
+function commerce_pos_currency_view_denominations($currency_id) {
+  $output = '';
+
+  // Get the denomination type info.
+  $denomination_type_info = commerce_pos_denomination_get_currency_types_from_id(array($currency_id));
+
+  if (!empty($denomination_type_info)) {
+    $denomination_type_info = $denomination_type_info[$currency_id];
+
+    $output .= l(t('Add Denomination'), 'admin/commerce/config/pos/denomination-type/' . $denomination_type_info->currency_id . '/denomination/add', array(
+      'attributes' => array('class' => array('add-denomination', 'button'))
+    ));
+
+    // Now, get the denominations for this denomination type.
+    $denominations = commerce_pos_currency_denomination_get_denominations(array($denomination_type_info->currency_code));
+    if (!empty($denominations)) {
+      $output .= t('<h3>Current Denominations for Type @type</h3>', array(
+        '@type' => $denomination_type_info->currency_code
+      ));
+
+      $output .= '<table class="commerce-pos-current-denominations">';

```

Comment:
It's preferable to use #theme => 'table' with $header and $rows in most cases, for consistency with the theme. But this is a settings page so up to you...